src/app/core/views/Preferences/tabs/Account/Language.tsx

```diff
@@ -133,6 +139,16 @@ export default function Language(): JSX.Element {
             >
               <p>{translate('lang.cn')}</p>
             </MenuItem>,
+            <MenuItem
+              key={'ru'}
+              onClick={() => {
+                setLang('ru');
+                i18next.changeLanguage('ru');
+                dayjs.locale(ru);
+              }}
+            >
+              <p>{translate('lang.ru')}</p>
+            </MenuItem>,
           ]}
```

Comment:
This is getting too long, to facilitate the maintenance of the code, better to change it for a component that, given an array with the different languages, generates as many `MenuItem` as needed.